src/components/privateHeader.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
File name should be capitalized

Reply:
in react they are not capitilized

Reply:
I would agree with you in often in React projects, files are not capitalized, like [this](https://github.com/codeforboston/windfall-elimination/tree/develop/src/components). I think this helps prevent conflicts between collaborators with git on operating systems that distinguish capitals and those that do not. I just had to clean up a challenge like this on another repo. But having capitals in file names has unfortunately been the convention we've followed in this one since the tutorial we started with.